src/api/java/CMakeLists.txt

```diff
@@ -2,28 +2,112 @@
 ## CMakeLists.txt
 ## Top contributors (to current version):
 ##   Mudathir Mohamed
-## This file is part of the CVC4 project.
+## This file is part of the CVC5 project.
 ## Copyright (c) 2009-2021 by the authors listed in the file AUTHORS
 ## in the top-level source directory and their institutional affiliations.
 ## All rights reserved.  See the file COPYING in the top-level source
 ## directory for licensing information.
 ##
 
-# create a directory for the cvc package
-file(MAKE_DIRECTORY "${CMAKE_CURRENT_BINARY_DIR}/cvc")
+# create directories
+file(MAKE_DIRECTORY "${CMAKE_CURRENT_BINARY_DIR}/src/main/java/cvc5/")
```

Comment:
If we stored the Java files in `src/api/java/cvc5` would Java complain about the folder structure?

Reply:
I guess  `src/api/java/cvc5` can work, but my I understand generated files should be in the `build` directory. Also files in the build directory  are not tracked by git, so generated files would not appear with `git status`.

Reply:
Note that the prefix is still `CMAKE_CURRENT_BINARY_DIR`, which is within the build directory.

Reply:
Oh, I misunderstood this with the  main source files. Named `main` to `api`.